### DISCUSSION TITLE
Speed up index conformation time

### DISCUSSION THREAD

**Discussion by pascalgottret** [photoprism/photoprism#2750] [Q&A]
Hello, I use photoprism for a big event (100 persons) where everyone is able to share their photos on the cell phones. I use the public mode so nobody has to enter a password. 

However, it takes a lot of time to index the photos after uploading. 
Is there a way to speed up the indexing process or even do it later automatically, so the user does not notice? 
I do not want the user to scare away. It must be simple, easy and not confusing.

thanks

**Comment by graciousgrey** [comment]
There is no way to delay indexing after web uploads. And you rather don't want this as it would confuse people even more if their photos don't show up in the UI (which is only possible after indexing).
You find performance tips here: https://docs.photoprism.app/getting-started/troubleshooting/performance/
